2010-12-03  Michael Tautschnig  <mt@debian.org>

	* setup-storage.8: Clarify that partition/volume sizes are integers, not
		decimal numbers.
Index: trunk/man/setup-storage.8
===================================================================
--- trunk.orig/man/setup-storage.8
+++ trunk/man/setup-storage.8	
@@ -378,9 +378,11 @@
 .br
          /* size in kilo (KiB), mega (default, MiB), giga (GiB), tera (TiB),
 .br
-          * petabytes (PiB) or percentage of disk size or RAM size;
+          * petabytes (PiB) or percentage of disk size or RAM size; integers
+.br
+          * only, no decimal numbers.
 .br          
-          * in future releases KB, MB, GB, ... will be treated as 1000 instead
+          * In future releases KB, MB, GB, ... will be treated as 1000 instead
 .br
           * of 1024 (KiB, MiB, GiB, ...) multipliers */
 
